src/mame/casio/pv1000.cpp

```diff
@@ -261,8 +259,23 @@ void pv1000_state::io_w(offs_t offset, uint8_t data)
 		m_sound->voice_w(offset, data);
 	break;
 
+	case 0x04:
+		/* Bit 1 = Matrix IRQ enabled */
```

Comment:
MAME preference is C++ style // for single-line comments and /* */ for multi-line.  You're mostly following that already, just a few exceptions.